packages/happo-target-react-native/src/defaultOptions.js

```diff
@@ -0,0 +1,96 @@
+const path = require('path');
+
+const iosAppPath = path.join(
+  __dirname, // path of this file
+  '../', // root of this module
+  'runner/ios/build/Build/Products/Debug-iphonesimulator/HappoRunner.app',
+);
+
+const androidAppPath = path.join(
+  __dirname, // path of this file
+  '../', // root of this module
```

Comment:
Totally agree about not using `__dirname`. If you have a better idea of how to do this, can you let me know?  I couldn't think of any...